src/components/Carousel/Carousel.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+import * as styles from "./Carousel.styles";
+
+import fix1 from "../../assets/img/fix-1.jpg";
+import fix2 from "../../assets/img/fix-2.jpg";
+import fix3 from "../../assets/img/fix-3.jpg";
+import fix4 from "../../assets/img/fix-4.jpg";
+
+const Carousel = () => {
+  const images = [fix1, fix2, fix3, fix4];
```

Comment:
이미지는 외부에서 불러올 수 있게 props로 설정해줘야할 것 같아요(공통 컴포넌트 -> 재사용가능해야함)
타입스크립트에서는 props로 넘길때 인터페이스 설정까지 필수 !!
```suggestion
interface CarouselProps {
  images: string[];
}

const Carousel = ({ images }: CarouselProps) => {
```